src/main/groovy/az/pashabank/controller/StudentController.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Created By Nadir Jabbarli
+ * Date : 7/5/2022
+ * Time : 10:58 AM
+ * Project Name : student-service
+ */
+
+package az.pashabank.controller;
+
+import az.pashabank.model.StudentEntity;
+import az.pashabank.repository.StudentRepository;
+import az.pashabank.service.impl.StudentServiceImpl;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping(path = "/students")
+@CrossOrigin(origins = "*")
+public class StudentController {
+    private final StudentServiceImpl studentService;
+
+    private final StudentRepository studentRepository;
+
+    public StudentController(StudentServiceImpl studentService, StudentRepository studentRepository) {
+        this.studentService = studentService;
+        this.studentRepository = studentRepository;
```

Comment:
why do you need studentRepository in here?

Reply:
Done